google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -157,7 +164,7 @@ def update(
         self,
         description: Optional[str] = None,
         labels: Optional[Dict[str, str]] = None,
-        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+        request_metadata: Sequence[Tuple[str, str]] = (),
```

Comment:
For my education, why is this required? What is the request_metadata btw?

Reply:
This is where the `request_metadata` will be used:

https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform_v1/services/featurestore_service/client.py#L911

When we used `request_metadata: Optional[Sequence[Tuple[str, str]]] = None`, it will throw an error, 

https://github.com/googleapis/python-aiplatform/blob/main/google/cloud/aiplatform_v1/services/featurestore_service/client.py#L993

so we use the same default value '()' as in the GAPIC layer.